tests/test_loader.py

```diff
@@ -61,7 +67,7 @@ def test_load_item_using_default_loader(self):
         il.add_value('name', u'marta')
         item = il.load_item()
         assert item is i
-        self.assertEqual(item['summary'], u'lala')
+        self.assertEqual(item['summary'], [u'lala'])
```

Comment:
I think this line technically makes the PR backward incompatible. I believe this is a reasonable price to pay, considering that the current code breaks if one wants to add more values to the same field, i.e. adding `il.add_value('summary', u'foobar')` after creating this loader results in
```
ERROR: test_load_item_using_default_loader (__main__.BasicItemLoaderTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_loader.py", line 62, in test_load_item_using_default_loader
    il.add_value('summary', u'foobar')
  File "/.../scrapy/loader/__init__.py", line 79, in add_value
    self._add_value(field_name, value)
  File "/.../scrapy/loader/__init__.py", line 95, in _add_value
    self._values[field_name] += arg_to_iter(processed_value)
TypeError: can only concatenate str (not "list") to str